build.sh

```diff
@@ -52,8 +52,8 @@ else
 	echo Using existing version file.
 fi
 
-# For OSX compatibility >= 10.7
-MACOSX_DEPLOYMENT_TARGET=10.7
+# For OSX compatibility >= 10.8
+MACOSX_DEPLOYMENT_TARGET=10.8
```

Comment:
That's due to a recent bump in dmd, isn't it? Seems unrelated to the PR though, at least it's uncommented why it's here.

Reply:
It causes linker warnings because linked dependencies are built targeting 10.8.

Reply:
See previous comment: https://github.com/dlang/dub/pull/1433#discussion_r179668984.